tutorials/cp-starter-isuite-onboard-subscribe/cp-starter-isuite-onboard-subscribe.md

```diff
@@ -116,7 +116,7 @@ In this step, you authorize a user to access the Integration Suite Provisioning
 
     !![Add Capabilities](5.1.1Addcapapbilities.png)
 
-    !![Additional functions selection](5.2.1Addfunctions_new.png)
+    <!---!![Additional functions selection](5.2.1Addfunctions_new.png)--->
```

Comment:
If you don't need this line, please remove it to keep your code clean.

Reply:
Hi @MichaelCzcz , 
Its still in discussion. 

Hence I commented. 

We may require this in near future.